.github/workflows/release.yml

```diff
@@ -0,0 +1,248 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        description: 'Release type'     
+        required: true
+        default: 'auto' 
+        type: choice
+        options:
+          - auto
+          - minor
+          - snapshot.0
+          - snpashot.1
+          - edge 
```

Comment:
I think this indentation for lists is not what we are currently using in other YAML.
I think @jmazzitelli knows better, since he has a lot of YAML in the operator side.

That said, I'm not sure if indentation here should match the opeator's one.

Reply:
We typically indent everything in the list at the same level as the parent. 
```yaml
parent:
- starts on same level
- again on same level
```

We don't further indent items in our operator yaml.

I don't know if we care to keep that format the same as this release.yml - but if you are using an editor with indentation rules, it would be nice to have it set with the same format rules and not have to change based on the file.

Reply:
I formatted the YAML documents with a Red Hat YAML plugin on VS Code. But if we are using other identation, I can change but I'll wait for @jmazzitelli comments to confirm.

Reply:
@jmazzitelli yes, I will modifiy the identation, no problem.